### DISCUSSION TITLE
expandir el TX EN TODAS LAS BANDAS

### DISCUSSION THREAD

**Discussion by delta1trieta** [egzumer/uv-k5-firmware-custom#67] [Ideas]
Para mi gusto le faltaria una opción en el menú secreto de expansión del TX en todas las bandas. Aun quedan equipos que trabajan en 80 mhz y con este walkie y su correspondiente antena se podria utilizar. Por favor tengalo en cuenta para futuras actualizaciones. Gracias por su gran trabajo.

**Comment by ErikS-web** [comment]
See [issues/68](https://github.com/egzumer/uv-k5-firmware-custom/issues/68)
